music_assistant/server/providers/snapcast/__init__.py

```diff
@@ -305,3 +314,15 @@ async def _get_empty_stream(self) -> str:
             if "id" in new_stream and new_stream["id"] not in used_streams:
                 return new_stream["id"]
             port += 1
+
+    def _player_state(self, player_id: str) -> PlayerState:
```

Comment:
I would name this function "_get_player_state" if you want to be consistent. Below is a function named "_set_childs_state"